### PR TITLE
Enable interactions and:

### PR DESCRIPTION
**Pull request by kajahno**
# Summary

* Refactor to use es6 syntax
* Add all commands to build and package in package.json
* Leave default interactions from discordjs example
* Configure babel and update documentation
* add commands as cron
* refactor previous leetcodedailymessage command

In addition this makes adding more commands easier.

Completes #21

### CODE REVIEW

**Comment by Not4Now** [comment]
Please solve conflicts and then re-request my review

**Comment by kajahno** [comment]
@Not4Now done

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "targets": {
+          "node": "current"
+        }
+      }]
+    ],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-proposal-object-rest-spread"
+    ]
+}
```
Comment (Not4Now) [inline .babelrc:13]: Please add a new line

**Review by Not4Now — APPROVED** [review]
LGTM

**Pull request merged**